snapshot_function/packages/latest-snapshot/link/link.js

```diff
@@ -0,0 +1,44 @@
+const fetch = require('node-fetch');
+const { XMLParser, XMLBuilder, XMLValidator } = require("fast-xml-parser");
+
+const region = 'fra1'
+const bucket = 'forest-snapshots'
+
+const base_url = 'https://' + region + '.digitaloceanspaces.com/' + bucket;
+
+// "calibnet/forest_snapshot_calibnet_2022-11-30_height_81393.car"
+
+async function main(args) {
+    // default to compressed snapshots unless we're told otherwise
+    let compressed = !(args.compressed === 'false' || args.compressed === 'no' || args.compressed === '0');
+
+    // default to the calibnet network unless we're told otherwise
+    let network = args.network || 'calibnet';
+
+    const response = await fetch(base_url);
+    const body = await response.text();
+
+    const parser = new XMLParser();
+    let jObj = parser.parse(body);
```

Comment:
what is `jObj`? Could we use a more explicit name?

Reply:
Renamed to `s3_listing`.